tox.ini

```diff
@@ -1,9 +1,8 @@
 [tox]
-envlist = py27,py33,py34,py35,pypy,pypy3
+envlist = py27,py34,py35,py36,pypy,pypy3
```

Comment:
It did in the commits from #5. But it doesn't say anything about test coverage yet.